/camera_node:
  ros__parameters:
    image_save_path: '/home/kami/kami_coder_github/kamibot/images/'
    source_frame: map
    target_frame: base_footprint
    use_sim_time: true